docs/0900|Sync_With_Database.md

```diff
@@ -0,0 +1,106 @@
+## Syncing to Database Recipie with Firebase 
```

Comment:
I think this should just be `Syncing to Firebase`

Reply:
Changed to `Syncing to Firebase`